getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,285 @@
+---
+layout: default
+title: Installing Community Using Containers
+published: true
+sorting: 80
+---
+
+The instructions in this guide describe how to download and install the latest version of CFEngine Community in a Docker containerized environment using pre-compiled rpm packages and ubi9 images.
+
+This guide describes how to set up a client-server model with CFEngine and, through policy, manage both containers.
+
+Docker containers will be created, one container to be the Policy Server (server), and another container that will be the Host Agent (client).
+
+Both the containers will run **_ubi9-init_** images and communicate on a container network.
+Upon completion, you are ready to start working with CFEngine.
+
+
+## Requirements
+* 1G+ disk space
+* 1G+ memory
+* Working [Docker Engine](https://docs.docker.com/engine/) or [Podman](https://podman.io/) setups on a supported [x86_64](https://en.wikipedia.org/wiki/X86-64) platform.
+
+**Note**: This document considers [Docker Engine](https://docs.docker.com/engine/) for all examples.
+Use of [Podman](https://podman.io/) shall be similar with adequate adaptations. (_Ref_: [Emulating Docker CLI with Podman](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman)).
+
+## Overview
+1. Installing container engine
+2. Preparing CFEngine hub in container
+3. Preparing CFEngine host in container
+4. Using docker compose
+    1. Preparing container image for CFEngine
+    2. Using docker compose service
+5. Glossary
+6. References
+
+## Installing container engine
+**Ref**: [Install Docker Engine](https://docs.docker.com/engine/install/)
+
+OR
+
+**Ref**: [Podman Installation Instructions](https://podman.io/docs/installation)
+(_Optionally_: [Emulating Docker CLI with Podman](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman))
+
+## Preparing CFEngine hub in container
+Run the container with systemd
+
+```command
+docker run --privileged -dit --name=cfengine-hub registry.access.redhat.com/ubi9-init /usr/sbin/init
+```
+
+Prepare the container for **cfengine-hub**
+
+```command
+docker exec cfengine-hub bash -c "dnf -y update; dnf -y install procps-ng iproute"
+```
+
+Install cfengine-community package
+
+```command
+docker exec cfengine-hub bash -c "dnf -y install https://cfengine-package-repos.s3.amazonaws.com/community_binaries/Community-3.24.0/agent_rhel9_x86_64/cfengine-community-3.24.0-1.el9.x86_64.rpm"
```

Comment:
Here I think you should mention our quick install script instead so that  the instructions always install the latest LTS instead of a hard-coded version.

I will adjust your steps when I test to use the quickinstall script and make a suggestions.